src/test/java/uk/gov/companieshouse/service/AppealServiceTest.java

```diff
@@ -65,20 +66,22 @@
 
     @Test
     void testCreateAppeal_returnsResourceId() {
+        ReasonEntity reasonEntity = createReasonEntityWithOther();
+        Reason reason = createReasonWithOther();
+        when(appealMapper.map(any(Appeal.class))).thenReturn(createAppealEntity(null, reasonEntity));
+        when(appealRepository.insert(any(AppealEntity.class))).thenReturn(createAppealEntity(TestData.Appeal.id, reasonEntity));
 
-        when(appealMapper.map(any(Appeal.class))).thenReturn(createAppealEntity(null));
-        when(appealRepository.insert(any(AppealEntity.class))).thenReturn(createAppealEntity(TestData.Appeal.id));
-
-        assertEquals(TestData.Appeal.id, appealService.saveAppeal(createAppeal(), TestData.Appeal.CreatedBy.id));
+        assertEquals(TestData.Appeal.id, appealService.saveAppeal(createAppeal(reason), TestData.Appeal.CreatedBy.id));
```

Comment:
This could be simplified to:

```
Appeal appeal = createAppeal(createReasonWithOther());
AppealReason appealReason = createAppealEntity(createReasonEntityWithOther());

when(appealMapper.map(appeal).thenReturn(appealReason);
when(appealRepository.insert(appealEntity)).thenReturn(appealReason);

assertEquals(TestData.Appeal.id, appealService.saveAppeal(appeal));
```

Reply:
Test constants were used to simplify the unit tests. Refactoring to remove unnecessary assertions has been added into a unit test refactoring task. This will be resolved separately to this PR.